mtcpclient/reporting.go

```diff
@@ -38,3 +38,45 @@ func StartBackgroundReporting(numberConnections int, rinterval int) (chan tcpcli
 
 	return connStatusCh, connStatusTracker
 }
+
+func FinalMetricsReport(connectionDescriptions []tcpclient.Connection) string {
+	mr := calculateMetricsReport(connectionDescriptions)
+	return "Time to establish TCP connections min/avg/max/stdDev = " +
```

Comment:
Given processing time of errored connections is also considered (see also my comment below on this), I'd recommend to change this to "TCP processing time" (as this figure may represent time to get an error, not only successfully stablished connections)